packages/consts/src/consts.ts

```diff
@@ -111,7 +116,25 @@ export const USERNAME = {
 
     // Regex matching a potentially allowed username. The numbers must match MIN and MAX!
     // Note that username must also pass isForbiddenUser() test to be allowed!
-    REGEX: /^[a-zA-Z0-9_.-]{3,30}$/,
+    REGEX: /^[a-zA-Z0-9_-]{3,30}$/,
+};
+
+export const EMAIL = {
+    MIN_LENGTH: 3,
+    MAX_LENGTH: 254, // see https://www.rfc-editor.org/errata_search.php?rfc=3696&eid=1690
+
+    REGEX: EMAIL_REGEX,
+};
+
+/**
+ * Profile name (such as organization or first / last name) constraints.
+ */
+export const PROFILE_NAME = {
+    MIN_LENGTH: 3,
+    MAX_LENGTH: 50,
+
+    // Regex to prohibit anything that could potentially form an URL or email address.
+    REGEX: /^[^/.@><]*$/,
```

Comment:
The originally proposed solution of excluding strings with '//:' in them did not work completely. Gmail and other email providers (tested also using Seznam and Yahoo) recognized also urls without the protocol and emails without the emailto protocol.

Excluding the dot, slash and @ effectively excludes any viable urls and email addresses. The > and < characters were previously checked and prohibited in custom function in apify-core (not sure what was the reason, probably something interpolation-wise 🤷).